libcontainer/rootfs_linux.go

```diff
@@ -144,11 +147,30 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		}
 		return nil
 	case "tmpfs":
+		tmpFilePath := ""
 		stat, err := os.Stat(dest)
 		if err != nil {
 			if err := os.MkdirAll(dest, 0755); err != nil {
 				return err
 			}
+		} else {
```

Comment:
I'm not sure this should be an `else`, because the point of the `if` is to create the mountpoint? Or am I misunderstanding what you're doing here?